distros/sandbox/gitea/statefulset-gitea.yaml

```diff
@@ -164,7 +167,8 @@ spec:
               port: http
             timeoutSeconds: 1
           resources: {}
-          securityContext: {}
+          securityContext:
```

Comment:
@vjayaramrh this is not clear here, you want a capability and `privileged` true flag?

Reply:
@arora-sagar Without the privileged: true flag, the below issue is seen and fails to bring up the gitea-0 container
```
[fedora@mgmt ~]$ kubectl logs gitea-0 -n gitea

Defaulted container "gitea" out of: gitea, init-directories (init), init-app-ini (init), configure-gitea (init)

s6-svscan: fatal: unable to open .s6-svscan/lock: Permission denied
```

Reply:
It will be better if you find the exact capability which is needed rather than giving `privileged`

Reply:
Good point

Reply:
@Sagar Arora do you have any suggestions on the linux capability that can be used other than `privilege: true`, I tried a couple (CAP_CHOWN, CAP_DAC_OVERRIDE, CAP_FOWNER, CAP_SETFCAP, CAP_SYS_ADMIN ) and have not been able to resolve the issue

Reply:
FYI Note that the /bin/s6-svscan is owned by root and run as user git, hence my guess is privileged: true is required. Please share any other suggestions you may have. 
```
[fedora@mgmt ~]$ kubectl exec gitea-0 -n gitea -c gitea -- ls -l /bin/s6-svscan
-rwxr-xr-x    1 root     root         22336 Jun 29  2022 /bin/s6-svscan
[fedora@mgmt ~]$ kubectl exec gitea-0 -n gitea -c gitea -- ps
PID   USER     TIME  COMMAND
    1 git       0:00 /bin/s6-svscan /etc/s6
   15 git       0:00 s6-supervise gitea
   16 git       0:00 s6-supervise openssh
   17 git       0:00 /usr/local/bin/gitea web
   18 root      0:00 sshd: /usr/sbin/sshd -D -e [listener] 0 of 10-100 startups
   86 git       0:00 ps

```

Reply:
@arora-sagar I am of the opinion that it maybe better to explore using the gitea/gitea:1.19.3-rootless than use gitea/gitea:1.19.3

Reply:
@arora-sagar Thanks for sharing, the one thing that we had to change to make it working in our custom scc was to use the below and it fixed the issue without having to make changes to the statefulset-gitea.yaml
```
runAsUser:
  type: RunAsAny
```
cc: @dkosteck